pkg/controller/releasemanager/releasemanager_controller.go

```diff
@@ -152,8 +153,9 @@ func newReconciler(mgr manager.Manager, c utils.Config) reconcile.Reconciler {
 }
 
 // add adds a new Controller to mgr with r as the reconcile.Reconciler
-func add(mgr manager.Manager, r reconcile.Reconciler) error {
+func add(mgr manager.Manager, r reconcile.Reconciler, c utils.Config) error {
 	_, err := builder.ControllerManagedBy(mgr).
+		WithOptions(controller.Options{MaxConcurrentReconciles: c.ConcurrentReleaseManagers}).
```

Comment:
:mind-blown:

Reply:
btw, the limiting factor on revision concurrency is DDOSing prom endpoint.